### PR TITLE
Added Sync option description to the docs

### PR DESCRIPTION
**Pull request by sidolov**
In the scope of Inventory pull request, https://github.com/magento/inventory/pull/2795 new configuration is introduced. 

The current pull request should fix a documentation gap for a newly introduced feature in the 2.3.5 Magento release.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/catalog/inventory-options-global.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ Configure the default configuration options for product and stock for your websi
         The message begins to appear when the quantity in stock reaches the threshold. For example, if set to `3`, the message “Only 3 left” appears when the quantity in stock reaches three. The message adjusts to reflect the quantity in stock, until the quantity reaches zero.
 
     - To display an “In Stock” or “Out of Stock” message on the product page, set **Display Products Availability In Stock on Storefront** to `Yes`.
+    
+    - To keep consistency between inventory and catalog, set **Synchronize with Catalog** to `Yes`. With the option enabled inventory data will be adjusted according to the catalog changes (product removed; product SKU changed; product type changed; etc.)
```
Comment (leslietilling) [inline src/catalog/inventory-options-global.md]:
```suggestion
    - To keep consistency between inventory and catalog, set **Synchronize with Catalog** to `Yes`. With this option enabled, inventory data will be adjusted according to the catalog changes (such as product removed, product SKU changed, and product type changed).
```

**Comment by sidolov** [comment]
@leslietilling these changes for an upcoming release - 2.3.5

**Comment by leslietilling** [comment]
running tests

**Pull request merged**